columnphysics/icepack_atmo.F90

```diff
@@ -942,6 +950,10 @@ subroutine icepack_atm_boundary(sfctype,                   &
 
       Cdn_atm_ratio_n = c1
 
+      if (trim(atmbndy) == 'default') then
+         atmbndy = 'similarity' ! For backward compability only ...
+      endif
+
```

Comment:
It's a good idea to keep backwards compatibility. 
But I *think* that we now tend to do this kind of treatment directly in `ice_init` (in CICE, probably in Icepack the equivalent would be in `icedrv_init`)...

Reply:
FYI. It is also done in CICE. So running CICE, you'll never ends up here - unless it is later changed in CICE.
If backward compatibility is only relevant for CICE, I'll suggest to remove this "default=similarity" from ICEPACK.